eng/common/pipelines/templates/archetype-typespec-emitter.yml

```diff
@@ -453,6 +453,30 @@ extends:
                 -PRLabels '$(PRLabels)'
               workingDirectory: $(Build.SourcesDirectory)
 
+        # Custom job to update http-client-csharp-mgmt package when http-client-csharp is published
```

Comment:
move this step to https://github.com/Azure/azure-sdk-for-net/blob/main/eng/packages/http-client-csharp/ci.yml instead